doc/RELEASE.rst

```diff
@@ -81,6 +81,10 @@ Test the upload by using pypi dev as a download source
 
 ::
 
+   virtualenv -p python3 labgrid-crossbar-release-<your-version-number>
```

Comment:
Why use the version number in the venv path?

Reply:
The other venv a few lines below does the same, that's why I've done the same here.